### DISCUSSION TITLE
Cannot add unity3d configuration to my consulo ide

### DISCUSSION THREAD

**Discussion by NwosuTy** [consulo/UnityEditorConsuloPlugin#31] [Q&A]
for some reason the consulo ide stopped recognizing unity3d scripts and i tried configuring it but i'm unable to see the unity3d template
![My project (4) - AnimatorHandler cs - Consulo 2021 12 13-Apr-22 11_41_52 AM](https://user-images.githubusercontent.com/94308549/163168126-c0686891-83ce-4b02-babf-4c0d92d704aa.png)
![My project (4) - Consulo 2021 12 12-Apr-22 9_29_02 AM](https://user-images.githubusercontent.com/94308549/163168135-2ad64078-fc19-45c6-8c32-472e7c9dc16d.png)

**Comment by VISTALL** [comment]
Hello. Looks like you have problem with project importing. Can you close project, remove .consulo dir from project, and open it via Open or Import Project action, and select project dir (not csproj, or etc).

You can make screenshots what you do, and I will say whats wrong
